src/dashboard/components/options/OptionsHome.js

```diff
@@ -9,10 +9,48 @@ import DarkOptionsSelect from './DarkOptionsSelect'
 
 export default function OptionsHome(props) {
   const [settings, _, saveSetting] = useSettingsInfo()
+  const [backupType, setBackupType] = useState('days')
 
   let backupLocation = settings.user.backupLocation
   if (!backupLocation || backupLocation == 'default') backupLocation = BACKUP_BASE_PATH
 
+  const renderDropdown = () => {
+    return (
+      <select onChange={(event) => setBackupType(event.target.value)}>
+        <option value="days">{t('Days of Backups')}</option>
+        <option value="number">{t('Number of Backups')}</option>
+      </select>
+    )
+  }
+  const displayBackupOption = () => {
+    switch (backupType) {
```

Comment:
I think the switch here is unnecessary, maybe we can use a ternary operator since the 2 <FormControl /> are almost the same

Something like:

```suggestion
      <div className="backup-type">
                <ControlLabel>{t('Number of Backups to Keep')}</ControlLabel>
                <FormControl
                  type="number"
                  value={backupType === 'number' ? settings.user.numberOfBackups : settings.user.backupDays}
                  defaultValue={30}
                  onChange={(event) => saveSetting(backupType === 'number' ? 'user.numberOfBackups':'user.backupDays', Number(event.target.value))}
                />
                <HelpBlock>{t('Backups beyond this will be erased')}</HelpBlock>
              </div>
```